docs/tutorial-react-native/start-streaming.mdx

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 3
+---
+
+# Start streaming
+
+How to stream your camera
+
+:::tip Enable devices before connecting
+
+You can enable your camera and microphone before calling connect method. 
+This way, you can show user camera preview. Once connect method is called,
+enabled camera and microphone will start streaming to Room.
```

Comment:
**suggestion:** 
```suggestion
You must enable your camera and microphone before calling the connect method if you want to stream video or audio. 
Once connected, the camera and microphone will start streaming to the Room.
```

Reply:
Are you sure that you have to enable microphone/camera before connecting? 🤔 We need to verify it, as think you can connect without these devices